apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/NPIOrgLookupIT.java

```diff
@@ -0,0 +1,19 @@
+package gov.cms.bfd.server.war;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import gov.cms.bfd.data.npi.lookup.NPIOrgLookup;
+import java.io.IOException;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+
+/** Integration tests for NPIOrgLookup. */
+public class NPIOrgLookupIT {
+  /** Verifies that it returns a valid npi org */
+  @Test
+  public void VerifyAValidNPIOrg() throws IOException {
```

Comment:
We should also have a negative test that checks for UNKNOWN too

Reply:
Done it return empty